x-pack/plugins/rule_registry/server/rule_registry/defaults/ilm_policy.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ILMPolicy } from '../types';
+
+export const defaultIlmPolicy: ILMPolicy = {
+  policy: {
+    phases: {
+      hot: {
+        actions: {
+          rollover: {
+            max_age: '90d',
+            max_size: '50gb',
```

Comment:
No change or anything, but just wanted to note the `.siem-signals*` ILM is [currently configured](https://github.com/elastic/kibana/blob/1216b0f7cd2c0697393547432bee343ed33b96b5/x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/signals_policy.json#L10) to `50gb/30d`. 


We haven't had much feedback or issue around this, but something for us to think about as @pmuellr mentioned with regards to mutating older alerts that might be in aged-out/frozen indices.

Reply:
For event log, we're using 50gb/30d as well, for basic rollover, with a delete phase at 90d.  I don't recall getting any customer feedback that this is too short / too long, yet.

https://github.com/elastic/kibana/blob/master/x-pack/plugins/event_log/server/es/documents.ts